Gemfile

```diff
@@ -30,6 +30,7 @@ gem "tailwindcss-rails"
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem "jbuilder"
 
+gem 'json', '~> 2.6', '>= 2.6.3'
```

Comment:
Even though the article I sent you is suggesting to add this entry to the `Gemfile`, it is not needed.

You can remove this and run `bin/bundle` so that `Gemfile.lock` be updated as well and you will see that everything will continue working fine.